xbmc/cores/AudioEngine/CMakeLists.txt

```diff
@@ -95,27 +95,28 @@ if(CORE_SYSTEM_NAME STREQUAL rbpi)
                       Sinks/AESinkPi.h)
 endif()
 
-if(CORE_SYSTEM_NAME STREQUAL darwin OR CORE_SYSTEM_NAME STREQUAL ios)
-  list(APPEND SOURCES Sinks/osx/AEDeviceEnumerationOSX.cpp
+if(CORE_SYSTEM_NAME STREQUAL darwin)
```

Comment:
ah ok, I though it was already changed so CORE_SYSTEM_NAME=osx
sorry for the noise.